app/experimenter/nimbus-ui/src/styles/index.scss

```diff
@@ -39,3 +39,29 @@ $sizes: (
 .font-weight-semibold {
   font-weight: 600;
 }
+
+.positive-significance {
+  color: green;
+  font-weight: bold;
+}
+
+.negative-significance {
+  color: red;
+  font-weight: bold;
+}
+
+.positive {
+  background-color: green;
+}
+
+.negative {
+  background-color: red;
+}
+
+.neutral {
+  background-color: grey;
+}
+
+.border-3 {
+  border-width:3px !important;
```

Comment:
My linter wants a space between the colon and `3px`. Do we need `!important` here? I know Bootstrap uses it so I wouldn't be surprised.

Reply:
Yes because the other border classes applied overwrite this `border-width` so `!important` prevents that